apps/scan/frontend/src/screens/voter_screen.tsx

```diff
@@ -33,16 +33,12 @@ export function VoterScreen({
     refetchInterval: POLLING_INTERVAL_FOR_SCANNER_STATUS_MS,
   });
 
-  // The scan service waits to receive a command to scan or accept a ballot. The
-  // frontend controls when this happens so that ensure we're only scanning when
-  // we're in voter mode.
-  const scanBallotMutation = scanBallot.useMutation();
+  // The backend waits to receive a command accept a ballot. The frontend
+  // controls when this happens for now. In the future, we should move this to
+  // the backend.
   const acceptBallotMutation = acceptBallot.useMutation();
   useQueryChangeListener(scannerStatusQuery, {
     onChange: (newScannerStatus) => {
-      if (newScannerStatus.state === 'ready_to_scan') {
-        scanBallotMutation.mutate();
-      }
       if (newScannerStatus.state === 'ready_to_accept') {
```

Comment:
We still trigger accept on the frontend because it may require voter interaction.

Reply:
But this particular case where we don't need voter interaction could be moved to the backend, right? (We don't necessarily need to)

Reply:
That's true. I didn't change it because I thought having it split between the frontend and backend might be more confusing. That said, I didn't try it. What do you think?